lua/nvim-web-devicons.lua

```diff
@@ -807,6 +807,10 @@ local icons = {
     color = "#89e051",
     name = "Zsh"
   };
+  ["terminal"] = {
+    icon = ""
+    color = "#31B53E",
+    name = "Terminal"
```

Comment:
Just a heads up missing `,` after the icon, and missing closing `};`

Reply:
woups, reviewing on smartphone is a bad idea ! Thanks for the note